blog/authors.yml

```diff
@@ -27,7 +27,7 @@ niewiarowski:
   image_url: https://github.com/andrzejniewiaro.png
 
 niedzwiedzka:
```

Comment:
Klucz można poprawić w pliku authors.yml

Reply:
Jeżeli poprawię klucz w `authors.yml`, to będę musiała poprawić też we wszystkich moich dotychczasowych tekstach. Trochę się wtedy tego robi.

Reply:
Masz rację, nie pomyślałem o tym. Może nie warto, jak uważasz. By the way, sorki za pomyłkę, bo to pewnie ja tak napisałem. 😅